compiler/ir/ir.tree/src/org/jetbrains/kotlin/ir/builders/ExpressionHelpers.kt

```diff
@@ -40,6 +40,9 @@ inline fun IrBuilderWithScope.irLetS(
     nameHint: String? = null,
     body: (IrValueSymbol) -> IrExpression
 ): IrExpression {
+    if (value is IrGetValue)
```

Comment:
Let's remove `irLet` above to avoid different behavior for a similar-looking function (it's unused both in Kotlin and kotlin-native)

Reply:
Done